notifications_utils/formatters.py

```diff
@@ -64,7 +64,7 @@
 
 whitespace_before_punctuation = re.compile(r'[ \t]+([,\.])')
 
-hyphens_surrounded_by_spaces = re.compile(r'\s+[-|–|—]{1,3}\s+')
```

Comment:
I think this regex was written with the idea that a pipe (`|`) acts as a OR, but it's counted as a character in reality.